### PR TITLE
Support setting label/environment keys via environment variables

### PR DESCRIPTION
**Pull request by rdsubhas**
Firstly, thanks for this awesome image. But we have one problem:
## Problem
- We use a custom env variable called `APP_NAME` to indicate name of the app
- So our target containers should be run with both `COLLECTD_DOCKER_APP_ENV=APP_NAME` as well as `APP_NAME=<our app name>`

This is very cumbersome. If all our apps are following a specific environment variable, then we should be able to set that value **globally** at this collectd-container level, rather than adding **two** environment variables for every app. I mean, if we are able to set additional variables to target containers, we may as well set the default `COLLECTD_APP_ENV` right?
## Solution
- You can now set an environment variable called `APP_ENV_KEY=<env variable that you use for apps>` when starting this `collectd` container. It will simply use this key to check all containers instead of the default `COLLECTD_DOCKER_APP`.
- Similarly, you can set `APP_LABEL_KEY`, `APP_ENV_KEY`, `TASK_LABEL_KEY`, `TASK_ENV_KEY` and those all will be used instead of the default `COLLECTD_` labels or environment variables.
- Basically configure things globally at this `collectd` container level instead of for each container
- The old way of overriding the label with `collectd_docker_app_label`, `collectd_docker_task_label`, `COLLECTD_DOCKER_APP_ENV`, `COLLECTD_DOCKER_TASK_ENV` still exists, so for some specific containers, you may set it to override the defaults as well.

### CODE REVIEW

**Comment by bobrik** [comment]
This looks great. Can you rename `Getenv` to `getenv`, andd add docs?

**Comment by rdsubhas** [comment]
@bobrik thanks! I have updated the `README` and changed the environment variables to be more consistent. Can you please have a look...

Hopefully the CI build failing was some momentary glitch, because these changes are completely backwards compatible. Waiting for the next build...

**Comment by bobrik** [comment]
There were no reason to change 80 char line length limit. Please revert this.

LGTM otherwise.

**Comment by rdsubhas** [comment]
@bobrik I have set it back to 80 char in the README.

The reason I changed it was, the `Environment variables` section of the README already did not follow 80 chars, and it was a bit cumbersome to enforce this there because of the one-line descriptions for each environment variable. Also Markdown doesn't care about single line wraps anyway. Because of the above two I removed the line wraps to be consistent with env variables and markdown. I have reverted that for now.

**Comment by bobrik** [comment]
Thanks!
